model.md

```diff
@@ -0,0 +1,164 @@
+# Model
+
+### <a name="struct"></a> Struct
+
+This is the primary object...
+
+| Field   | Type                      | Required | Notes                                                  |
+|---------|---------------------------|----------|--------------------------------------------------------|
+| name    | ref                       | true     | The common name used to identity the type of structure |
+| label   | string or [Label](#label) | true     | Describes the struct in the UI                         |
+| collectionLabel | string or [Label](#label) | true | Describes a collection of the struct in the UI     |
+| fields  | [Field](#field)[]         | true     |                                                        |
+| blocks  | [block](#block)[]         | true     | Array of [blocks](#block)                              |
+
+
+### <a name="field"></a> Field
+
+The property level portions of a [struct](#struct)
+
+| Field   | Type          | Required | Missing Val | Notes                                     |
+|---------|---------------|----------|------------ |-------------------------------------------|
+| name    | ref           | true     | N/A         | The common name used to describe identify the field |
+| label   | string or [Label](#label)| true | N/A  | Describes the field in the UI             |
+| type    | string        | true     | N/A         | Identifies the [Type](#type) of the field |
+| key_field | boolean     | false    | false       | Indicates that the field is used as the key or part of a composite key to identify a specific instance of a struct |
+| required| boolean       | false    | false       | Indicates that the field is required in order to be saved |
+| unique  | boolean       | false    | false       | The field value must be unique across all instances of the same struct |
+| initial_value | value cosistent with the field [type](#type) | false |  | The value used when an instance of the struct is created |
```

Comment:
typo? should `cosistent` be `consistent`?

Reply:
It should be consistent.